examples/bot.py

```diff
@@ -10,7 +10,10 @@
 # When you make a bot, we refer to it as the "client."
 # The client is the main object that interacts with the Gateway, what talks to Discord.
 # The client is also the main object that interacts with the API, what makes requests with Discord.
-client = interactions.Client("your bot token will go here.")
+client = interactions.Client(
+    "your bot token will go here.",
+    intents=interactions.Intents.DEFAULT | interactions.Intents.GUILD_MESSAGE_CONTENT,
```

Comment:
Can we make this just the default?

Would also add comments

Reply:
> Can we make this just the default?

Example bot has `on_message_create` event so we need message intent for this

Reply:
Would rather have an example just to show bare connection tbh, other example files should be considered

Reply:
I agree, a separate example about intents should be added

Reply:
Wait...., why it's `on_ready` not `on_start`?

Reply:
> Wait...., why it's `on_ready` not `on_start`?

The gateway can reconnect multiple times during the bot's life, and `on_ready` is triggered for each reconnection. However, `on_start` is triggered only **once** when the bot has been started for the first time, so it's better to use it.

Reply:
Dead.

Reply:
is time to resolving conversation

Reply:
Thoughts haven't changed: people would rather have a separate example page for intents, which would mean gutting out the `on_message_create` event handler code in total here.